### DISCUSSION TITLE
Add support for task list summary information

### DISCUSSION THREAD

**Discussion by TPS** [community/community#4222] [Mobile] [answered]
As shown on https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/about-task-lists,
- [x] ![task-list-summary.png](https://user-images.githubusercontent.com/185902/122692772-ed6d5680-d204-11eb-986f-766a091cbf4e.png) isn't supported in-app.

**Comment by brianlovin** [comment, marked as answer]
Thanks for the note @TPS – we have this on our roadmap, but I can't say exactly when we'll have this UI in the apps. Appreciate the nudge!
